great-idea-website/index.html

```diff
@@ -12,52 +13,92 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
-</head>
+  </head>
+<body>
+  <header>
 
-  <body>
+    <nav>
+      <a href="services"> Services </a>
```

Comment:
Make sure if you are calling a section on the same page to use a # before the section you are calling.